packages/apputils-extension/src/index.ts

```diff
@@ -595,7 +603,7 @@ export default plugins;
 /**
  * The namespace for module private data.
  */
-namespace Private {
+export namespace Private {
```

Comment:
We don't export the Private namespace (hence the name). If you need some functions out of it, we can move functions out of it.

Reply:
Although see the discussion in #6022 about whether we *should* be more liberal about exporting functionality from some packages.

Reply:
Thanks, I was thinking of that discussion too.

Reply:
Whoops! I left that in from some experimentation I was doing in response to @Carreau's post. I do lean towards his position. I'm a big fan of the typical Python approach: if a var starts with `_` you probably shouldn't touch it, but you can still access it if you need to.

Funnily enough, there is another example of `export namespace Private` that was committed to the code a couple of years ago [here](https://github.com/jupyterlab/jupyterlab/blob/master/packages/coreutils/src/settingregistry.ts#L1275). I guess its a bug?

Reply:
Yeah, that one looks inadvertent to me.

Reply:
back to proper Private